### ISSUE TITLE
5 (optional)

### ISSUE THREAD

**Issue by abisliouk** [abisliouk/IE675b-machine-learning#19]
Explore variants of preprocessing and logistic regression further. Suggestions include:
• Try gradient descent on the original data without using z-scores.
• Add a bias feature (make sure that you do not scale it).
• Try to reduce the training set size and compare MLE and MAP estimation (ideally using cross-validation).
• Run a logistic regression method from some existing library. Do you get the same results?
• Experiment with different gradient-based optimizers. To do so, we provide a PyTorch im- plementation, which allows you to quickly change the optimizer. For a list of optimizers, see here.
